[2022-11-06 14:08:09,328] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xetra_DAG.run_xetra_etl scheduled__2022-11-04T15:00:00+00:00 [queued]>
[2022-11-06 14:08:10,119] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xetra_DAG.run_xetra_etl scheduled__2022-11-04T15:00:00+00:00 [queued]>
[2022-11-06 14:08:10,121] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-06 14:08:10,122] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-11-06 14:08:10,123] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-06 14:08:10,167] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_xetra_etl> on 2022-11-04 15:00:00+00:00
[2022-11-06 14:08:10,184] {standard_task_runner.py:52} INFO - Started process 193 to run task
[2022-11-06 14:08:10,190] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'xetra_DAG', 'run_xetra_etl', 'scheduled__2022-11-04T15:00:00+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/xetra_DAG.py', '--cfg-path', '/tmp/tmpcn_seha0', '--error-file', '/tmp/tmpc996db1k']
[2022-11-06 14:08:10,193] {standard_task_runner.py:80} INFO - Job 165: Subtask run_xetra_etl
[2022-11-06 14:08:10,357] {task_command.py:369} INFO - Running <TaskInstance: xetra_DAG.run_xetra_etl scheduled__2022-11-04T15:00:00+00:00 [running]> on host 0163e0790208
[2022-11-06 14:08:10,636] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xetra_DAG
AIRFLOW_CTX_TASK_ID=run_xetra_etl
AIRFLOW_CTX_EXECUTION_DATE=2022-11-04T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-04T15:00:00+00:00
[2022-11-06 14:08:17,838] {python.py:173} INFO - Done. Returned value was: None
[2022-11-06 14:08:17,871] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=xetra_DAG, task_id=run_xetra_etl, execution_date=20221104T150000, start_date=20221106T140809, end_date=20221106T140817
[2022-11-06 14:08:17,924] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-06 14:08:18,005] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
